src/devices/ovrheadset/OVRHeadset.h

```diff
@@ -126,6 +126,7 @@ class OVRHeadset : public yarp::dev::DeviceDriver,
     std::map<ovrResult, std::string> error_messages;
 
     IFrameTransform* tfPublisher;
+    bool             relative;
```

Comment:
it will be.. but there are still some problem with it (basically we have to find a way to track the human torso).